static/i18n/en/app.json

```diff
@@ -619,7 +629,7 @@
       "step2": {
         "title": "Did you save your recovery phrase by yourself?"
       },
-      "step3": {
+      "step3":{
```

Comment:
prettier will not like that

Reply:
yeah I was looking at it, prettier doesnt douch this file though because its in `static`, change the script in package.json?